src/components/data/EntitiesContent.js

```diff
@@ -284,9 +284,8 @@ const EntitiesContent = ({
   } = useColumnProvenance(workspace, entityKey);
   const [showColumnProvenance, setShowColumnProvenance] = useState(undefined);
 
-  const buildTSV = (columnSettings, entities) => {
+  const buildTSV = (columnSettings, entities, isSet, forDownload) => {
     const sortedEntities = _.sortBy('name', entities);
-    const isSet = _.endsWith('_set', entityKey);
```

Comment:
Since `isSet` can be determined within this function, it could be removed from the function parameters. Then each caller wouldn't have to compute it.

Reply:
At least for downloading, the caller needs to know the value of `isSet` outside of `buildTsv`.  At some point I was also using it for copying; now I am not but seems like it's going to have to be computed twice anyway - unless I could pull it outside of all the functions.  It only depends on `entityKey` which I think is more widely available

Reply:
Since it's cheap to calculate, I think I'd vote for the occasional duplication over the additional plumbing to have all callers calculate it and pass it through.